en-US/E.xml

```diff
@@ -89,14 +89,17 @@
 			</listitem>
 
 		</varlistentry>
-		<varlistentry id="EOL">
+		<varlistentry id="eol">
 		 <term>EOL</term>
 			<listitem>
 			 <para>
```

Comment:
ok, you're going to hate me now... Can you add `<emphasis>` around n. and adj. to be consistent with the rest of the book? I know a few cases remain where this hasn't been updated but I try to fix at every opportunity.

Reply:
Done :)